--- crates/flocking/tests/havoc.rs
+++ crates/flocking/tests/havoc.rs
@@ -7,8 +7,7 @@
     #![proptest_config(ProptestConfig { failure_persistence: None, cases: 1000, .. ProptestConfig::default() })]

     #[test]
-    #[should_panic]
-    fn test_havoc_nan_propagation(
+    fn test_havoc_nan_propagation_fixed(
         v_x in prop_oneof![
             Just(f64::INFINITY),
             Just(f64::NEG_INFINITY),
